### PR TITLE
When deploying logging, storage config is stripped out of the imported ClusterLogging instance

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1734107

### CODE REVIEW

**Comment by mburke5678** [comment]
@ewolinetz PTAL

File: modules/efk-logging-deploying-about.adoc [hunk 1]
```diff
@@ -106,7 +106,7 @@ You can configure a persistent storage class and size for the Elasticsearch clus
       elasticsearch:
         nodeCount: 3
         storage:
-          storageClass:
+          storageClassName:
```
Comment (jcantrill) [inline modules/efk-logging-deploying-about.adoc]: it should be:
```
storageClassName: "gp2"
size: "200G"
```
no name key

File: modules/efk-logging-deploying-about.adoc [hunk 2]
```diff
@@ -106,7 +106,7 @@ You can configure a persistent storage class and size for the Elasticsearch clus
       elasticsearch:
         nodeCount: 3
         storage:
-          storageClass:
```
Comment (ewolinetz) [inline modules/efk-logging-deploying-about.adoc]: the whole structure should be
```
elasticsearch:
    storage:
        storageClassName: "gp2"
        size: "200G"
```

**Review by ewolinetz — CHANGES_REQUESTED** [review]
`storageClassName` and `size` are at the same level. there is no `name`

File: modules/efk-logging-deploying-about.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/efk-logging-deploying-about.adoc]: Ooops! Sorry!

File: modules/efk-logging-elasticsearch-storage.adoc [hunk 3]
```diff
@@ -30,9 +30,9 @@ metadata:
       elasticsearch:
         nodeCount: 3
         storage:
-          storageClass: 
+          storageClassName:
```
Comment (ewolinetz) [inline modules/efk-logging-elasticsearch-storage.adoc]: same edit here

**Comment by mburke5678** [comment]
@xltian PTAL or assign to someone who can review. Thank you!

**Comment by ewolinetz** [comment]
:+1:

**Comment by anpingli** [comment]
LGTM

**Review by anpingli — APPROVED** [review]

**Comment by mburke5678** [comment]
Verified in BZ

**Pull request merged**